android/lib/src/main/java/com/swmansion/gesturehandler/GestureUtils.java

```diff
@@ -3,49 +3,49 @@
 import android.view.MotionEvent;
 
 public class GestureUtils {
-    public static float getLastPointerX(MotionEvent event, boolean averageTouches) {
+    public static float getLastPointerX(GestureHandlerMotionEventAdapter event, boolean averageTouches) {
```

Comment:
move this to adapter class

also we can add `getRawX(int i)` to adapter and use it instead of the offset hack